_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -268,7 +268,7 @@ Element has ARIA role, but no ARIA states or properties
 
 `aria-checked` state with empty value
 
-**Note**: The HTML validator flags an `aria-checked` attribute with an empty value as an issue. However WAI-ARIA 1.1 indicates `aria-checked` has a default value of `undefined`.
+**Note**: The HTML validator flags an `aria-checked` attribute with an empty value as an issue. However, since WAI-ARIA 1.1, `aria-checked` has a default value of `undefined`.
```

Comment:
Should this reference be updated to 1.2?

Reply:
I don't think so. This is true since 1.1 and it's still true in 1.2, therefore "since 1.1" is correct, right?